### PR TITLE
try and fix OOM issues on Jenkins X infra running go builds

### PR DESCRIPTION
**Pull request by rawlingsj**

### CODE REVIEW

**Comment by jtnord** [comment]
Looks like a good starter and avoids premature optimisation until we know how this behaves in the wild
/approve

File: jx-build-templates/templates/jenkins-go-buildtemplate.yaml [hunk 1]
```diff
@@ -20,6 +20,8 @@ spec:
       value: /home/jenkins
     - name: DOCKER_CONFIG
       value: /home/jenkins/.docker/
+    - name: _JAVA_OPTIONS
+      value: "-Xmx 400m"
```
Comment (agentgonzo) [inline jx-build-templates/templates/jenkins-go-buildtemplate.yaml]:
```suggestion
      value: "-Xmx400m"
```

**Pull request merged**